AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/AWSCloudWatchLoggingCategoryClient.swift

```diff
@@ -0,0 +1,195 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AWSPluginsCore
+import Amplify
+import Combine
+import Foundation
+import AWSCloudWatchLogs
+import AWSClientRuntime
+import Network
+
+/// Concrete implementation of
+/// [LoggingCategoryClientBehavior](x-source-tag://LoggingCategoryClientBehavior)
+/// ensures the right log files and cloud watch streams are used according to
+/// the application's authentication state.
+///
+/// - Tag: CloudWatchLoggingCategoryClient
+final class AWSCloudWatchLoggingCategoryClient {
+    private var _enable: Bool = true
```

Comment:
🤔 why the underscore?

Reply:
This was to avoid naming collision with the public func `enable()`.  I've updated the internal variable to `enabled` in the latest rev.